.gitlab-ci.yml

```
@@ -39,17 +40,21 @@ test:static:
     before_script:
         - pip install --cache-dir=~/.cache/pip-test mypy
         - pip install --cache-dir=~/.cache/pip-test pylint
+        - pip install --cache-dir=~/.cache/pip-test .
         - pip install --cache-dir=~/.cache/pip-test -r requirements.txt
+        - pip install --cache-dir=~/.cache/pip-test -r tests/unit/requirements.txt
     script:
         - mypy src
         - pylint --rcfile=.pylintrc-develop src
+        - pylint --rcfile=.pylintrc-develop --disable=no-self-use --disable=no-member --disable=too-many-ancestors tests/unit/test_*.py
 
 test:unit:
     extends: .test
     stage: test
     image: python:3.6-buster
     before_script:
         - pip install --cache-dir=~/.cache/pip-test/ -r requirements.txt
+        - pip install --cache-dir=~/.cache/pip-test -r tests/unit/requirements.txt
```

nit pick for consistency
```suggestion
        - pip install --cache-dir=~/.cache/pip-test/ -r tests/unit/requirements.txt
```